packages/mobile/src/invite/reducer.ts

```diff
@@ -17,8 +17,7 @@ export const inviteReducer = (
   switch (action.type) {
     case REHYDRATE: {
       return {
-        ...state,
-        ...getRehydratePayload(action, 'invite'),
+        invitees: getRehydratePayload(action, 'invite').invitees ?? [],
```

Comment:
Only possible gotcha with this, is if we keep this reducer and someone else adds new props and that don't end up being persisted, if this isn't changed.
But we're going delete this reducer eventually, right?